setup.py

```diff
@@ -7,7 +7,9 @@
     install_requires = [
         "octopus==1.0.0",
         "esprit",
-        "Flask"
+        "Flask",
+        "setproctitle",
```

Comment:
One more thing - generally it's best to freeze library dependencies if they have versions published on PyPI. That should prevent the app from stopping to work in the future when somebody installs it just because it's 2019 and `psutil` is now version 310 and has a vastly different AI-powered API.